lib/salestation/web/input_validator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Salestation
+  class Web < Module
+    class InputValidator
+      include Deterministic::Prelude
+
+      def self.[](**validations)
```

Comment:
one * is for args, two is for keyword args. should be one here

Reply:
but these are keyword args, you call it like
```
Salestation::Web::InputValidator[
    accept: Salestation::Web::InputValidators::AcceptHeader['application/json', 'application/xml'],
    content_type: Salestation::Web::InputValidators::ContentTypeHeader['application/json'],
    my_value: lambda do |my_value|
      if my_value == 'foo'
        Deterministic::Result::Success(nil)
      else
        Deterministic::Result::Failure('invalid value')
      end
    end
  ]
```

Reply:
ah my bad